src/rules/prettier.ts

```diff
@@ -36,8 +36,8 @@ export default class Husky {
     // TODO
   }
 
-  exists() {
-    return this.isInDevDep();
+  shouldBeApplied() {
+    return !this.isInDevDep();
```

Comment:
Why ?

Reply:
The shouldBeApplied returns true only if the rule should be applied, so only if prettier is not in devDependencies